addons/website_sale_stock/models/product_product.py

```diff
@@ -11,7 +11,7 @@ class ProductProduct(models.Model):
     cart_qty = fields.Integer(compute='_compute_cart_qty')
 
     def _compute_cart_qty(self):
-        website = getattr(request, 'website', None)
+        website = request and hasattr(request, 'website') or None
         if not website:
             return
         cart = website.sale_get_order()
```

Comment:
Here you will have a `AttributeError: 'bool' object has no attribute 'sale_get_order'` because you have a change of value of request to boolean in this PR.

Reply:
Fixed in 4f03956

Thanks

Reply:
It's not fixed, you will have the same error `AttributeError: 'bool' object has no attribute 'sale_get_order'`

```python
request = 'cadena'
>>> website = request and hasattr(request, 'upper') or None
>>> cart = website.sale_get_order()
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    cart = website.sale_get_order()
AttributeError: 'bool' object has no attribute 'sale_get_order'
```

Reply:
Thank you @moylop260 

Fixed at my last commit change